connectors/bitkeep.ts

```diff
@@ -0,0 +1,26 @@
+import LockConnector from '../src/connector';
```

Comment:
You don't need this file, this is handled automatically for injected connectors.